sdc/tests/test_series.py

```diff
@@ -4782,6 +4782,69 @@ def test_series_pct_change_impl(S, periods=1, fill_method='pad', limit=None, fre
         msg = 'Method pct_change(). The object periods'
         self.assertIn(msg, str(raises.exception))
 
+    def test_series_setitem_for_value(self):
```

Comment:
It seems that all three positive tests only cover integer series case, so floats and strings are not covered.
It would be good to add them too, you can write an aggregated test for diff data types/values, like here:
https://github.com/IntelPython/sdc/blob/abfa12bb622f3ca248c2513479487895b0e2d769/sdc/tests/test_series.py#L2176